nima_core/nima_consciousness_core.py

```
@@ -0,0 +1,797 @@
+        
+        return self.dream_engine.get_dream_report(session)
+    
+    def measure_current_phi(self) -> Optional[PhiMeasurement]:
+        """
+        Measure Φ for the most recent workspace entry.
+        
+        Returns:
+            PhiMeasurement or None if no entries
+        """
+        if not self.config.enable_phi_tracking:
+            raise RuntimeError("Phi tracking is disabled in config")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)